### PR TITLE
Add binaries from Google Drive

### PR DESCRIPTION
**Pull request by GreatEmerald**
Obsoletes pull #164. This shows as able to automatically merge from GitHub.

### CODE REVIEW

File: Binaries.csv [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
 Animations,UT3NightshadeAnims.ukx,https://cloud.greatemerald.eu/s/5pAhIZsmcsmmXt7/download,876447b1d5f9bca5fc589981b88a5223b58e6e8f781c1607fb1aca945f5273bd,
+Animations,UT3PaladinAnims.ukx,https://drive.google.com/uc?export=download&id=1c5lknHX3QvrfAccpvxneiM6f_6jf2Ipv,b609d22f3de275c6f946bc42164e84670f60b0b214a327ffdffcfefbd8fbe9d8,--content-disposition
+Animations,UT3StealthbenderAnims.ukx,https://drive.google.com/uc?export=download&id=1Zi_iCWQuKAau2R3kfhPGz9lY5y-eZedZ,93f76efa9e1d4ba63f99f5eb9abc6043992eb3efeba1fc25a1cfc542f139b766,--content-disposition
+Animations,UT3ViperAnims.ukx,https://drive.google.com/uc?export=download&id=16N9Yx25K3qy_-SEERs_4AEy634sJtcL2,218a01b314dbf2d8dcd43a1a9b35cbe030adace209d2474de8798190e14ecb5a,--content-disposition
+Animations,UT3VH_Cicada_Anims.ukx,https://drive.google.com/uc?export=download&id=1DZCJnLy7IRMpJzHNN8loy-05YxL0lbm0,1bbe6bf21be961abb950b6f0a1bf4e3edbbf5998386ad26e656859e8b92e08cf,--content-disposition
+Animations,UT3VH_Hellbender_Anims.ukx,https://drive.google.com/uc?export=download&id=1UVMWr1TS6UR3CvhCeTNkxDUJLrUm0wFS,6b112be3e0a4a35a5353a99770396e9bbc1182f931534f02ab191db836cc89f9,--content-disposition
```
Comment (GreatEmerald) [inline Binaries.csv]: @HellDragon-HK The checksum does not match here. Check what's going on here.

File: Binaries.csv [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+Animations,UT3VH_Leviathan_Anims.ukx,https://drive.google.com/uc?export=download&id=1_2foPegkhCEcJ2hvyxMGEF7kD0fNqJSa,9ebd933daf9ad450d8c54235ca0cebb36f180fa5acc6741dbc4e53a29090c78e,--content-disposition
+Animations,UT3VH_Paladin_Anims.ukx,https://drive.google.com/uc?export=download&id=1zWJwZNHbxcwGCQ82hxSqzlI4NuJWgRoU,679f078381ae82854483e6c3cd8c9bd1d573f65a6a18a102019a3325ecb71c3f,--content-disposition
+Animations,UT3VH_Raptor_Anims.ukx,https://drive.google.com/uc?export=download&id=1GAGc8gKLVdO66snC_5j2LVyGBn9QxSTj,6294f557cb1f06e02c1973fbea67478b55242d3c95a61688c380178ad5ac43d1,--content-disposition
 Animations,UT3VH_Scorpion_Anims.ukx,https://drive.google.com/uc?export=download&id=1QSeKP6KxkT9vBNyptGE7ouPl76W1EL-C,b66bc22d1df53ae1335c21f5053c08fcb2af7c4554f829a64b3a2ea8618f6b2c,--content-disposition
+Animations,UT3VH_SPMA_Anims.ukx,https://drive.google.com/uc?export=download&id=1cDv2_FcH72_1rgIjBt5zdTU_pSRpkrqe,563c43204035a855062f92175c50f62807addd98c4e5a450fb4646918850f1ff,--content-disposition
```
Comment (GreatEmerald) [inline Binaries.csv]: @HellDragon-HK The checksum does not match here either.

**Comment by HellDragon-HK** [comment]
Just re-upload it maybe?

File: Binaries.csv (hunk 1 above)
Comment (HellDragon-HK) [reply Binaries.csv]: Which two here?

Comment (GreatEmerald) [reply Binaries.csv]: `UT3VH_Hellbender_Anims.ukx`. The comment is always about the line at the very bottom of the box.

**Comment by GreatEmerald** [comment]
Well, depends on which version is the correct one. If you have an old version and the file has a hash of the new one, then yes, upload it. But normally we'd version it by never overwriting files, rather uploading them anew and changing the download URL and hash.

**Comment by HellDragon-HK** [comment]
Looks like I missed uploading the current changes I have on the Hellbender....oh right the turret animations

**Comment by Unre-Alex** [comment]
I'll comment here because the other discussion is closed.
Textures are not finished, HellDragon has some outdated files. When they are done I'll upload them to google drive and make a pull request.

**Comment by HellDragon-HK** [comment]
@Unre-Alex I have outdated really? What all have you done since UT3VehicleTexturesDec21st?

**Comment by Unre-Alex** [comment]
Many small changes, mostly to normal maps. You don't need updated textures for the vehicles to work, there's no need to upload them now.

**Comment by HellDragon-HK** [comment]
Not to give the wrong vibe but that's kind of unfair for @100GPing100 isn't it?

**Comment by GreatEmerald** [comment]
@Unre-Alex OK, good to know, thanks!

@HellDragon-HK Yeap, now it works fine.

**Comment by Unre-Alex** [comment]
Well, if he wants an update I can always send him a link.

**Comment by 100GPing100** [comment]
Oh no, no need for an update. At most I run the game for only 5 minutes at a time to test changes and check for (new) bugs. Can't wait to see new juicy textures though!

**Pull request merged**